OSPSuite.PKParameters.xml

```diff
@@ -4,47 +4,47 @@
+    <PKParameter name="C_trough_tD2" displayName="C_trough_tD2" mode="Multi" dimension="Concentration (molar)" description="Trough concentration just before the second dose is administered"/>
+    <PKParameter name="C_trough_tDLast" displayName="C_trough_tDlast" mode="Multi" dimension="Concentration (molar)" description="Trough concentration just before the last dose is administered"/>
     <PKParameter name="C_tEnd" displayName="C_tEnd" mode="Single" dimension="Concentration (molar)" description="Concentration at the end of simulation" />
 
-    <PKParameter name="AUC" displayName="AUC_tEnd" mode="Single" dimension="AUC (molar)" description="Area under the concentration vs. time curve from the first to the last data point" />
-    <PKParameter name="AUC_norm" displayName="AUC_tEnd_norm" mode="Single" dimension="AUC (mass)" description="Area under the concentration vs. time curve from the first to the last data point normalized to dose, with dose in drug mass per body weight" />
+    <PKParameter name="AUC_tEnd" displayName="AUC_tEnd" mode="Single" dimension="AUC (molar)" description="Area under the concentration vs. time curve from the first to the last data point" />
+    <PKParameter name="AUC_tEnd_norm" displayName="AUC_tEnd_norm" mode="Single" dimension="AUC (mass)" description="Area under the concentration vs. time curve from the first to the last data point normalized to dose, with dose in drug mass per body weight" />
+
+    <PKParameter name="AUC_inf" displayName="AUC_inf" mode="Single" dimension="AUC (molar)" description="Area under the concentration vs. time curve from the first data point extrapolated to infinity" />
+    <PKParameter name="AUC_inf_norm" displayName="AUC_inf_norm" mode="Single" dimension="AUC (mass)" description="Area under the concentration vs. time curve from the first data point extrapolated to infinity normalized to dose, with dose in drug mass per body weight" />
 
-    <PKParameter name="AUC_inf"  displayName="AUC_inf" mode="Single" dimension="AUC (molar)" description="Area under the concentration vs. time curve from the first data point extrapolated to infinity" />
-    <PKParameter name="AUC_inf_norm"  displayName="AUC_inf_norm" mode="Single" dimension="AUC (mass)" description="Area under the concentration vs. time curve from the first data point extrapolated to infinity normalized to dose, with dose in drug mass per body weight" />
+    <PKParameter name="AUC_tD1_tD2" displayName="AUC_tD1-tD2" mode="Multi" dimension="AUC (molar)" description="Area under the concentration vs. time curve from the administration time of the first dose until the administration time of the second dose"/>
+    <PKParameter name="AUC_tD1_tD2_norm" displayName="AUC_tD1-tD2_norm" mode="Multi" dimension="AUC (mass)" description="Area under the concentration vs. time curve of the first application interval normalized to dose, with dose in drug mass per body weight"/>
```

Comment:
I assume "AUC_tD1_tD2" and "AUC_tD1_tD2_norm" is renaming of "AUC_t1_t2" and "AUC_t1_t2_norm"? (just the diff is not optimally showed by GitHub)

Reply:
In that case you can click view change and show a side by side.

* I have also changed all t1 parameter to be tD1
* I have also changed all t2 parameter to be tD2
* AUC => Auc_tEnd
* Fixed a typo intervall=>interval

If we want Cmin and Tmin to appear in PKSim and MoBi, we need to add the definition of those parameters here

If we want RLin to appear in PKSim and MoBi, we need to add the definition of those parameters here as well

Reply:
>If we want Cmin and Tmin to appear in PKSim and MoBi, we need to add the definition of those parameters here

>If we want RLin to appear in PKSim and MoBi, we need to add the definition of those parameters here as well

Let's skip both for the moment